src/requirements.txt

```diff
@@ -1,26 +1,26 @@
 -i https://pypi.python.org/simple
-asgiref==3.6.0 ; python_version >= '3.7'
-boto3==1.26.69
-botocore==1.29.69 ; python_version >= '3.7'
-cachetools==5.3.0
-certifi==2022.12.7 ; python_version >= '3.6'
+asgiref==3.7.2 ; python_version >= '3.7'
+boto3==1.26.144
+botocore==1.29.144 ; python_version >= '3.7'
+cachetools==5.3.1
+certifi==2023.5.7 ; python_version >= '3.6'
 cfenv==0.5.3
 cffi==1.15.1
-charset-normalizer==3.0.1 ; python_version >= '3.6'
-cryptography==39.0.1 ; python_version >= '3.6'
+charset-normalizer==3.1.0 ; python_full_version >= '3.7.0'
+cryptography==41.0.1 ; python_version >= '3.7'
 defusedxml==0.7.1 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-dj-database-url==1.2.0
+dj-database-url==2.0.0
 dj-email-url==1.0.6
-django==4.1.6
+django==4.2.1
```

Comment:
At a framework level, probably a good idea to check on Django's release notes:

- https://docs.djangoproject.com/en/4.2/releases/4.2/
  - There's some backwards incompatible features [here](https://docs.djangoproject.com/en/4.2/releases/4.2/#backwards-incompatible-changes-in-4-2), related to database

Reply:
I don't see usages of `allows_group_by_pk` (makes sense, this was a MySQL thing, we're using postgres) or `allows_group_by_selected_pks` in our codebase.